syntax = "proto3";

import "blockchain.proto";
import "request_response.proto";

service NodeService {
  rpc GetBlocks(BlocksRequest) returns (stream Block);
  rpc GetMempool(Empty) returns (stream Transaction);
  rpc GetMempoolSize(Empty) returns (MempoolSizeMessage);
};
